generator/app.go

```diff
@@ -457,6 +460,41 @@ func createGroupReadme(groups []Group, prefix string) error {
 	return nil
 }
 
+func createAnnualReportIssue(groups []Group, prefix string) error {
+	// figure out if the user wants to generate one group
+	var selectedGroupName *string
+	if envVal, ok := os.LookupEnv("WHAT"); ok {
+		selectedGroupName = &envVal
+	}
+
+	for _, group := range groups {
+		if !strings.HasPrefix(group.Dir, "sig-") {
+			continue
+		}
+
+		// skip generation if the user specified only one group
+		if selectedGroupName != nil && !strings.HasSuffix(group.Dir, *selectedGroupName) {
+			fmt.Printf("Skipping tmp/%s.md\n", group.Dir)
```

Comment:
if you moved `outputPath := ...`  to the top of the loop you could dry up your logging
```suggestion
			fmt.Printf("Skipping %v\n", outputPath)
```